### ISSUE TITLE
tag bug on iPhone

### ISSUE THREAD

**Issue by damienkeating** [owlstanding-labs/weekly-exhibition#11]
![img_9812](https://cloud.githubusercontent.com/assets/3681983/12245626/a4437f3e-b8a1-11e5-9b13-f7a9b9ac6d7d.PNG)

**Comment by speed-of-light** [comment]
This bug should belongs to [owlstanding-labs/website]
We can fix it in that project or build a feature for this project.

**Comment by damienkeating** [comment]
yep, fix it on that project then.

**Comment by damienkeating** [comment]
@speed-of-light Hi Nelson, can you fix this on [owlstanding-labs/website]?

**Comment by speed-of-light** [comment]
yes, but there are higher priority issues there, which one should we solve first?
